src/sagemaker_containers/environment.py

```diff
@@ -679,3 +689,24 @@ def model_server_workers(self):  # type: () -> int
         """Returns:
             (int): Number of worker processes the model server will use"""
         return self._model_server_workers
+
+
+@contextlib.contextmanager
+def temporary_directory(suffix='', prefix='tmp', dir=None):  # type: (str, str, str) -> None
```

Comment:
maybe `tmp_dir` or `tmpdir` for the method name instead? or `create_tmpdir` if you're worried about overusing 'tmpdir'